packages/UI/src/components/atoms/Link/StyledLink.ts

```diff
@@ -0,0 +1,77 @@
+import styled from 'styled-components';
+import theme from '../../../styles/theme';
+import { typographyStyles } from '../Typography';
+
+const lightColor = {
+  default: theme.colors.BLUE,
+  visited: theme.colors.BLUE_700,
+  hover: theme.colors.BLUE_600,
+  focus: theme.colors.BLUE_600,
+  focusBorder: theme.colors.LIGHT_BLUE_200,
+  active: theme.colors.BLUE_600,
+  disabled: theme.colors.GREYSCALE_CHARCOAL,
+};
+
+const darkColor = {
+  default: theme.colors.LIGHT_BLUE,
+  visited: theme.colors.LIGHT_BLUE_500,
+  hover: theme.colors.LIGHT_BLUE_200,
+  focus: theme.colors.BLUE_600,
+  focusBorder: theme.colors.LIGHT_BLUE_200,
+  active: theme.colors.BLUE_100,
+  disabled: theme.colors.GREYSCALE_GREY,
+};
+
+export const StyledLink = styled.a`
+  ${typographyStyles.pSmall}
```

Comment:
@alston-hsu 
I think the typography variant of link isn't yet defined so we have to create these [Typography variants](https://www.figma.com/file/EwzuhhvTulvFRMvhTD5VAh/Dev-Launchers-Style-Guide?node-id=1044%3A1054&t=O439lgOvsitp3B4S-4)
![image](https://user-images.githubusercontent.com/27841994/209867963-9584adac-bd45-43f5-8444-cbb25ac171fd.png)
`typographyStyles.pSmall` is a bit different than them